problem-26.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+// Stores “Hello World” to the text file and then shows content of the file on terminal after reading
+
+int main()
+{
+    FILE *file;
+
+    file = fopen("text-files/hello.txt", "w");
+    fprintf(file, "Hello World");
+    fclose(file);
+
+    file = fopen("text-files/hello.txt", "r");
+
+    if (file == NULL)
+    {
+        printf("Failed to open the file.\n");
+        return 1;
+    }
+
+    int maxCharsToRead = 100;

```

Comment:
Any reason why we are reading 100 characters specially if we are sure that the "Hello World" have less characters?